### DISCUSSION TITLE
Luxtorpeda integration

### DISCUSSION THREAD

**Discussion by TheEpicNoobZilla** [Heroic-Games-Launcher/HeroicGamesLauncher#1372] [Ideas]
Will you consider adding luxtorpeda as an option to launch linux games from GOG or Epic by luxtorpeda which enables native version for some games through source ports. This would benefit games like doom, quake or Stalker series (for the time of writing mainly Call of Pripyat and Clear Sky) since the only way to launch those ports is doing manually and since HGS already simplifes installing and running games for non windows users that does not have the steam copy of game

**Comment by noticons** [comment]
I've been waiting for this!
